app/controllers/task_lists_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class TaskListsController < ApplicationController
+  def index
+    @task_lists = TaskList.where(hidden: false).order('created_at ASC')
+  end
+
+  def show
+    @task_list = TaskList.find(params[:id])
+    @tasks = Task.where(task_list_id: @task_list.id)
+  end
+
+  def new; end
```

Comment:
I'm okay with doing that in follow up PR but we should start
implementing these methods, so they'll do something actually